lib/plugins/physics.js

```diff
@@ -280,19 +319,15 @@ function inject (bot, { physicsEnabled }) {
 
     if (bot.supportFeature('teleportUsesOwnPacket')) {
       bot._client.write('teleport_confirm', { teleportId: packet.teleportId })
-      // Force send an extra packet to be like vanilla client
-      sendPacketPositionAndLook(pos, newYaw, newPitch, bot.entity.onGround)
     }
     sendPacketPositionAndLook(pos, newYaw, newPitch, bot.entity.onGround)
 
     shouldUsePhysics = true
-    bot.entity.timeSinceOnGround = 0
```

Comment:
Why was this removed? Was it moved somewhere else?

Reply:
At the time, it was unused and wasn't specified in the API.

Reply:
ok